src/viur/core/modules/user.py

```diff
@@ -126,7 +126,7 @@ class UserSkel(skeleton.Skeleton):
     access = SelectBone(
         descr=i18n.translate("viur.user.bone.access", defaultText="Access rights"),
         values=lambda: {
-            right: i18n.translate("server.modules.user.accessright.%s" % right, defaultText=right)
+            right: i18n.translate(f"server.modules.user.accessright.{right}", defaultText=right)
```

Comment:
Hmm this looks awkward, as "server." is referred. Anyway, just leave it as is for now.